### PR TITLE
Add prion to CircleCi

### PR DESCRIPTION
**Pull request by MocanaAtBroad**
Looks like it is now deploying correctly!

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -12,17 +12,17 @@ references:
...
+      "CMI-OSTEO cmi-brain cmi-brain ANGIO cmi-mbc testboston cmi-mpc PRION"
   config_deployment_dir: &config_deployment_dir
       config/deployment
 
 # Container for all the builds
 executors:
   build-executor:
     docker:
-      - image: broadinstitute/study-website-build:05-07-2020A
+      - image: broadinstitute/study-website-build:07-28-2020A
```
Comment (MocanaAtBroad) [inline .circleci/config.yml:25]: Rebuilt image build image. For some reason gcloud utility was crashing.

**Comment by charlottedye** [comment]
When this goes through, how would I do the deployment of the frontend?  Would I just go into Circle CI and click something?

**Comment by fat23cat** [comment]
Am I right that we are not going to add nightly, conditionally, etc. jobs for prion for now?

**Comment by MocanaAtBroad** [comment]
> Am I right that we are not going to add nightly, conditionally, etc. jobs for prion for now?

I had forgotten to add the nightly! Updated now.

**Comment by MocanaAtBroad** [comment]
> When this goes through, how would I do the deployment of the frontend? Would I just go into Circle CI and click something?

3 ways to deploy:
1) Whenever we merge to `develop`
2) Nightly job from `develop`
3) Executing local script:
`build-utils/startbuilddeploy.sh prion BRANCH`
Option 3 is the one I have been using this morning to try out my changes. Of course, only works after you have committed and pushed changes to github.

**Review by charlottedye — APPROVED** [review]

**Pull request merged**